ПР2

Megrief opened the pull request:
Завершена верстка экрана поиска, добавлена поддержка темной темы, логика для EditText и TextWatcher, кнопки сброса поискового запроса, savedInstanceState. 
По ошибке сделал merge предыдущего ПР со всеми новыми коммитами. Не нашёл: как откатить уже закрытый ПР.

Megrief committed:
Реализована строка поиска, завершена верстка экрана поиска.

Megrief committed:
Реализована строка поиска, завершена верстка экрана поиска.

Megrief committed:
Delete .idea directory

Megrief committed:
Реализована строка поиска, завершена верстка экрана поиска.

Megrief committed:
Текст сообщений вынесен в строковые ресурсы.

Megrief committed:
Merge remote-tracking branch 'origin/dev' into dev

ilshat-abdulin commented:
> Завершена верстка экрана поиска, добавлена поддержка темной темы, логика для EditText и TextWatcher, кнопки сброса поискового запроса, savedInstanceState. По ошибке сделал merge предыдущего ПР со всеми новыми коммитами. Не нашёл: как откатить уже закрытый ПР.

Пришлось оставить комментарии (есть что исправить) к твоему коду, по текущему спринту, в смерженном PR. Обрати на это внимание, что лучше соблюдать порядок сдачи работы на проверку. Так проверки будут выполняться качественнее)
Надеюсь на понимание)

Megrief committed:
Исправлены недочеты:
Добавлен Toolbar на всех экранах.
Исправлена верстка.
Дополнена логика ClearButton на экране поиска.
Переработан SwitchCompat.
